lib/cc/analyzer/formatters/plain_text_formatter.rb

```diff
@@ -42,11 +42,11 @@ def engine_running(engine, &block)
           end
         end
 
-        def failed(output)
-          spinner.stop("Failed")
-          puts colorize("\nAnalysis failed with the following output:", :red)
+        def errored(output)
+          spinner.stop("Errored")
+          puts colorize("\nAnalysis errored with the following output:", :red)
           puts output
-          exit 1
+          exit 2
```

Comment:
Are you saying an error is `2` and reserving issues found as `1`?

Looking at http://tldp.org/LDP/abs/html/exitcodes.html#EXITCODESREF

> 1	Catchall for general errors
> 2	Misuse of shell builtins (according to Bash documentation)
>
> According to the above table, exit codes 1 - 2, 126 - 165, and 255 [1] have special meanings, and should therefore be avoided for user-specified exit parameters

I think it's fine to use `1` for general errors. For issues-found, it looks like 3-125 or 166-255 is available; I'd prefer going above 165, myself.